packages/client/src/checkout/types/getCheckoutOrderResponse.types.ts

```diff
@@ -11,11 +11,11 @@ export enum OrderStatusError {
 
 export type GetCheckoutOrderResponse = {
   id: number;
-  checkoutOrder?: CheckoutOrder;
-  shippingOptions?: ShippingOption[];
+  checkoutOrder: CheckoutOrder;
```

Comment:
I believe these properties could be optional since we could request a checkout order with a `fields` query, like `checkoutOrder,paymentMethods,shippingOptions,deliveryBundles,userPaymentTokens`.

Reply:
You're right! Fixed 😉